README.md

```diff
@@ -8,9 +8,15 @@
 
 # ![logo](./docs/images/logo_arlon.svg)
 
-Arlon is a lifecycle management and configuration tool for Kubernetes clusters.
-It allows an administrator to compose, deploy and configure a large number of
-*workload clusters* in a structured, predictable way.
+Arlon is a declarative, gitops based fleet management tool for Kubernetes clusters.
+It allows administrators to 
+- Deploy and upgrade a large number of *workload clusters* 
```

Comment:
```suggestion
- Deploy and upgrade a large number of *workload clusters* + applications
```

Reply:
@cruizen  bullet number 3 here addresses the application part, so I would rather skip it from this line